mysql-plugin/src/e2e-test/features/mysqlsource/DesignTimeValidation.feature

```diff
@@ -141,3 +141,73 @@ Feature: MySQL Source - Design time validation scenarios
       | referenceName  |
       | database       |
       | importQuery    |
+
+  Scenario: To verify MySQL source plugin validation error message with invalid reference test data
```

Comment:
All these validation scenarios should have required tag so that they run as part of required tests as well

Reply:
Done.